pkg/db/v1alpha2/db_init.go

```diff
@@ -18,7 +18,6 @@ func (d *dbConn) DBInit() {
 		parallel_trial_count INT,
 		max_trial_count INT,
 		status TINYINT,
-		metrics_collector_type TEXT,
```

Comment:
Are we not keeping the metric collector template in experiments table?

Reply:
We should. Added